src/tools/rbd/action/Kernel.cc

```diff
@@ -312,6 +313,14 @@ static int do_kernel_map(const char *poolname, const char *imgname,
     }
   }
 
+  r = krbd_is_image_mapped(krbd, poolname, imgname, snapname, mapped_info, img_mapped);
+  if(r < 0) {
+    std::cerr << "rbd: warning: can't get image map infomation: " << cpp_strerror(r) << std::endl;
+  }
+  else if (img_mapped){
```

Comment:
Nit: to keep a unified style, the braces should be on the same line:  `} else if (){`